VERSION

```diff
@@ -1 +1 @@
-1.0.0
+1.1.0
```

Comment:
1.1.1-SNAPSHOT?

Reply:
SNAPSHOT is how maven does things. If you look at the Jenkinsfile, merges on the develop branch have this syntax
`<version>-<commitid>` so its effectively the same thing.

Reply:
still needs to be 1.1.1 no, develop should be ahead of master

Reply:
ok ill change to snapshot since its more clear